bot/management/commands/request_matches.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+
+from common.utils import get_slack_client
+from matchmaking.models import TopicChannel
+from bot.processors.matchmaking import request_match_blocks
+
+
+class Command(BaseCommand):
+    help = """Send out onboarding questionnaire to set of users specified by a list of slack names. If no users
+    are specified then all users that don't have a row in the database will be contacted. There is a safety switch in
+    the script requiring you to start the send after it lists who will get the onboarding."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slack-team',
```

Comment:
The default gets all Slack teams, so it actually makes me more sane in the short term to have no default ID, because then I don't have to know the IDs for my QA env for testing vs the production env id.